lib/hawkular/alerts/alerts_api.rb

```diff
@@ -523,8 +523,24 @@ def to_h
   # server and not 'rubyfied'. So 'alertId' and not 'alert_id'
   # Check http://www.hawkular.org/docs/rest/rest-alerts.html#Alert for details
   class Alert < OpenStruct
+    attr_accessor :lifecycle
+
     def initialize(alert_hash)
       super(alert_hash)
+      @lifecycle = alert_hash['lifecycle']
+    end
+
+    def ackBy
+      statusBy('ACKNOWLEDGED')
+    end
+
+    def resolvedBy
+      statusBy('RESOLVED')
+    end
+
+    def statusBy(status)
+      a = @lifecycle.nil? ? [] : @lifecycle.select { |l| l['status'].eql? status }
+      a.empty? ? '' : a.last['user']
```

Comment:
After a detailed inspection, It seems that it was returning null before this change. 
See ackBy on: https://github.com/hawkular/hawkular-client-ruby/pull/156/files#diff-82e612a2fe3edccd5616886ec158d0cdL105